data_collections/internships.py

```
@@ -0,0 +1,256 @@
+        - link: The link to the application.
+        - entryDate: Date that the entry entered our `runningCSV.csv`
+        
+    Raises:
+        RuntimeError: If there's an error fetching or parsing the markdown file.
+    """
+    try:
+        response = requests.get(url, timeout=30)
+        response.raise_for_status()
+        markdown_content = response.text
+    except Exception as e:
+        raise RuntimeError(f"Failed to fetch the markdown file from {url}: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)